xml/System.Windows.Forms/BindingContext.xml

```diff
@@ -204,7 +204,15 @@ this->BindingContext->get_Item(dataSet1->Customers)->Position = 1;
         <param name="dataSource">The <see cref="T:System.Object" /> associated with the <see cref="T:System.Windows.Forms.BindingManagerBase" />.</param>
         <param name="listManager">The <see cref="T:System.Windows.Forms.BindingManagerBase" /> to add.</param>
         <summary>Adds the <see cref="T:System.Windows.Forms.BindingManagerBase" /> associated with a specific data source to the collection.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method is obsolete and unused.
```

Comment:
Here and elsewhere - Since the `<remarks>` information does not show up in Intellisense in Visual Studio, I think we should move this text to the `<summary>`, although I am not sure what is the best guidance for mentioning obsoletion. What would you suggest, @gewarren?

Reply:
@RussKie There are several methods in `BindingContext.cs` that are called out as obsolete in the remarks. Should we decorate them with `[Obsolete]` so the docs get generated correctly?